core/src/main/java/com/amplitude/core/Amplitude.kt

```diff
@@ -320,12 +355,17 @@ open class Amplitude internal constructor(
      * @return the Amplitude instance
      */
     @JvmOverloads
-    fun setGroup(groupType: String, groupName: String, options: EventOptions? = null): Amplitude {
+    fun setGroup(
```

Comment:
Did the lint settings change? Seems like a lot of style updates

Reply:
Yeah, I wonder if it is for the ktlint version installed is different than before..Would you prefer to revert it back?

Reply:
No preference on the style as long as it stays consistent. Just wanted to check if this was expected.